app/queries/get_application_choices_for_providers.rb

```diff
@@ -19,7 +19,7 @@ def self.call(providers:)
     ApplicationChoice.includes(*includes)
       .where('courses.provider_id' => providers, 'courses.recruitment_cycle_year' => RecruitmentCycle.current_year)
       .or(ApplicationChoice.includes(*includes)
-        .where('courses.accredited_provider_id' => providers))
+        .where('courses.accredited_provider_id' => providers, 'courses.recruitment_cycle_year' => RecruitmentCycle.current_year))
```

Comment:
I'd imagine `GetApplicationChoicesForProviders` to be called for previous cycles as well. Should this be a parameter, set by the controller?

Reply:
It will be soon

Reply:
When we implement this: https://manage-applications-beta.herokuapp.com/switch-cycle